semantic-conventions/src/tests/semconv/model/test_correct_parse.py

```diff
@@ -669,6 +735,12 @@ def semantic_convention_check(self, s, expected):
         self.assertEqual(expected["n_constraints"], len(s.constraints))
         self.assertEqual(expected["attributes"], [a.fqn for a in s.attributes])
 
+    def metric_check(self, metrics: Tuple[MetricSemanticConvention.Metric], expected):
+        for i, metric in enumerate(metrics):
```

Comment:
You can probably use `zip(metrics, expected)` instead of `enumerate` here.

Reply:
Yeah that's cleaner, thanks for the suggested. I went ahead and implemented it.